doc/src/modules/core.rst

```diff
@@ -46,9 +46,29 @@ singleton
 
 S
 ^
-.. autoclass:: S
+.. autoclass:: Singleton
    :members:
 
+.. note:: What's the difference between `x == 1` and `x is S.One`
+
+   1. `S.One` denotes a singleton object in SymPy.`x` is `S.One` compares its address 
+   with the address of the object denoted by `x`,so it takes practically no time to execute.
```

Comment:
```
compares its address with the address
```
This might be a little confusing for non CS oriented people. Perhaps it can be rephrased.